packages/dns-discovery/src/index.ts

```diff
@@ -15,6 +15,39 @@ export { NodeCapabilityCount };
 
 const log = debug("waku:peer-discovery-dns");
 
+const publicKey = "AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM";
+const fqdn = "test.waku.nodes.status.im";
+const enrTree = `enrtree://${publicKey}@${fqdn}`;
```

Comment:
Var split is not necessary, just save it in one string and it'll help with cspell